### DISCUSSION TITLE
Payment Gateway Problem

### DISCUSSION THREAD

**Discussion by netlinktech** [hotspotbilling/phpnuxbill#90] [Q&A]
Hey Boss, Assalamualaikum Warahmatullahi Wabarakatuh. I was facing a 2 problem on phpnuxbill, one is when i use bkash payment gateway first time there was no problem, problem shown when i want to create secend transection there is the problem transection failed massage is shown. 2nd one is if i host the application on cpanel api port not working. please give me some solution for this if u understand my massage.

![Screenshot_64](https://github.com/hotspotbilling/phpnuxbill/assets/62032355/ec1aa0ca-e5fe-4bad-b805-fd41c2038dc0)
![Screenshot_65](https://github.com/hotspotbilling/phpnuxbill/assets/62032355/115bd4cd-9647-4c5b-8ea6-723d6df89abf)

Thanks,
Best Regurds,
Mohammad Raju.

**Comment by lemowa** [comment]
Any solution for fix failed to create transaction sir ?
